### ISSUE TITLE
[Bug]: Incorrect operator (== instead of ===) + accessibility warning + unused imports

### ISSUE THREAD

**Issue by mangeshwagh91** [codeaashu/DevDisplay#1222]
### What happened?

A type-checking bug was found due to the use of the == operator instead of === in IndividualJourney.js, leading to incorrect comparisons. Additionally, an accessibility warning occurred because the iframe in VoiceSearch.jsx lacked a title attribute. Multiple unused imports across various components increased ESLint warnings, affecting code quality and readability. These issues collectively caused unnecessary warnings and potential unexpected behavior in the application.

### Add screenshots

src\components\Search\VoiceSearch.jsx
  Line 23:6:  React Hook useEffect has a missing dependency: 'handleListen'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

### What browsers are you seeing the problem on?

Chrome, Other, Brave, Microsoft Edge, Safari, Firefox

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by Shailaja-poojari** [comment]
Heyy there, I’d love to contribute to this project! Could you please assign me any open issue that needs attention?

**Comment by Arnav-Agarwal-01** [comment]
I would like work on this. I have previously contributed to this repo also. Please assign this to me @codeaashu , thanks

**Comment by Arnav-Agarwal-01** [comment]
> Hi there! This issue is still open. We are looking forward to your response. Assignees: None

Assign it to me then, if you can. 
Thanks :)
